pages/mesosphere/dcos/services/dcos-monitoring/1.4.0/operations/grafana/storage/index.md

```diff
@@ -0,0 +1,38 @@
+---
+layout: layout.pug
+navigationTitle: Grafana Storage
+title: Grafana Storage
+menuWeight: 20
+excerpt: Grafana Storage
+render: mustache
+model: ../../../data.yml
+---
+
+# Configuring Grafana Storage
+
+## DSS Volume
+
+You can configure Grafana to use a dedicated volume provided by the [DC/OS Storage Service](/services/storage/latest/) (DSS).
+
+At least one volume must be available for Grafana to use.
+For more information on creating volumes, see the [DC/OS Storage Service](/services/storage/latest/cli-references/dcos-storage-volume/dcos-storage-volume-create) documentation.
+
+When configuring Grafana, select `MOUNT` for its volume `type`, choose the minimum `size` for the volume in MB, and enter the `profile` for the volume.
+For more information on volume profiles, see the [DC/OS Storage Service](/services/storage/latest/cli-references/dcos-storage-profile/) documentation.
+
+The following configuration will deploy Grafana to a volume with a profile of `fast` that's 512 MB or larger.
```

Comment:
"that's" -> "that is"

Reply:
No, I changed all of this because linter told me to do so.

@cmastr Please fix linter!